packages/itwin/tree-widget/src/components/utils/UseTreeTransientState.ts

```diff
@@ -22,7 +22,7 @@ import { useControlledTreeLayoutStorage } from "@itwin/components-react";
  * }
  * ```
  *
- * @public
+ * @internal
  */
 export function useTreeTransientState<T extends Element>() {
   const { ref, persist, restore } = useControlledTreeLayoutStorage<T>();
```

Comment:
This is ControlledTree specific so it can be removed.

Reply:
removed